apps/graphql/src/apps/location/types/inputs/LocationsByTermInput.js

```diff
@@ -1,12 +1,23 @@
 // @flow
 
-import { GraphQLInputObjectType, GraphQLNonNull, GraphQLString } from 'graphql';
+import {
+  GraphQLInputObjectType,
+  GraphQLNonNull,
+  GraphQLString,
+  GraphQLList,
+} from 'graphql';
+
+import GraphQLLocationType from '../enums/LocationType';
 
 export default new GraphQLInputObjectType({
   name: 'LocationsByTermInput',
   fields: {
     term: {
       type: GraphQLNonNull(GraphQLString),
     },
+    types: {
+      description: 'desired location type output',
```

Comment:
Please just switch first description letter to capital `D` to be consistent with other descriptions.
Otherwise looks and works perfectly.

Reply:
🚀